### DISCUSSION TITLE
Nuovo Dongle Usb Sonoff

### DISCUSSION THREAD

**Discussion by ster1um** [indomus/forum#1020] [ZigBee, Z-Wave, Thread, Matter]
Oggi ho ricevuto la nuova usb dongle sonoff basata su cc2652p.
Non ho ancora avuto tempo di verificare il suo funzionamento...
Qualcuno la sta usando?

**Comment by essegi65** [comment]
Seguo anche io con interesse, tra l'altro lo si potrà usare anche senza RPI attaccato (alimentato da una presa)?

**Reply by ster1um** [reply]
> 
> 
> Seguo anche io con interesse, tra l'altro lo si potrà usare anche senza RPI attaccato (alimentato da una presa)?

assolutamente no, è un coordinator zigbee!

**Reply by essegi65** [reply]
Si lo so. ma ci sono dei coordinator zigbee che possono tranquillamente essere usati staccati dal RPI (anche dopo averli flashati con altri FW) magari questo non sarà il caso del sonoff però ne volevo avere certezza ;)

**Reply by ster1um** [reply]
> Si lo so. ma ci sono dei coordinator zigbee che possono tranquillamente essere usati staccati dal RPI (anche dopo averli flashati con altri FW) magari questo non sarà il caso del sonoff però ne volevo avere certezza ;)

Mi fai un esempio di coordinator zigbee senza WiFi che si possono scollegare dal hub ?

**Reply by snakuzzo** [reply]
che intendi "senza wifi" ? Che ci fai senza connettivià ?
Comunque dai un'occhiata alla [Z-Bee Duo](https://indomus.it/?s=Z-bee+duo) progettata e costruita dal nostro @gio-dot che ti permette di rendere indipendente il coordinator dall'hub...

E' tra gli adapter raccomandati da zigbee2mqtt -> [LINK](https://www.zigbee2mqtt.io/guide/adapters/#recommended)

Sulla pagina del progetto trovi tutti i dettagli -> [LINK](https://github.com/Gio-dot/Z-Bee-Duo)

P.S. Quella nuova di itead è fondamentalmente una copia "limitata" rispetto a quella creata da Giovanni mesi fa

**Reply by gio-dot** [reply]
> > Si lo so. ma ci sono dei coordinator zigbee che possono tranquillamente essere usati staccati dal RPI (anche dopo averli flashati con altri FW) magari questo non sarà il caso del sonoff però ne volevo avere certezza ;)
> 
> Mi fai un esempio di coordinator zigbee senza WiFi che si possono scollegare dal hub ?

La mia chiavetta Z-Bee si può collegare alla usb oppure tramite moduli wifi o ethernet opzionali si può usare dislocata da un altra parte rispetto a dove hai Homeassistant:https://github.com/Gio-dot/Z-Bee-Duo

**Reply by ster1um** [reply]
> > > Si lo so. ma ci sono dei coordinator zigbee che possono tranquillamente essere usati staccati dal RPI (anche dopo averli flashati con altri FW) magari questo non sarà il caso del sonoff però ne volevo avere certezza ;)
> > 
> > 
> > Mi fai un esempio di coordinator zigbee senza WiFi che si possono scollegare dal hub ?
> 
> La mia chiavetta Z-Bee si può collegare alla usb oppure tramite moduli wifi o ethernet opzionali si può usare dislocata da un altra parte rispetto a dove hai Homeassistant:https://github.com/Gio-dot/Z-Bee-Duo

Buonasera,
Apprezzo molto i suoi lavori e ritornando in merito quello che scrivi è proprio quello che intendevo. 
Vedi la risposta @snakuzzo

**Reply by segaura** [reply]
> > > Si lo so. ma ci sono dei coordinator zigbee che possono tranquillamente essere usati staccati dal RPI (anche dopo averli flashati con altri FW) magari questo non sarà il caso del sonoff però ne volevo avere certezza ;)
> > 
> > 
> > Mi fai un esempio di coordinator zigbee senza WiFi che si possono scollegare dal hub ?
> 
> La mia chiavetta Z-Bee si può collegare alla usb oppure tramite moduli wifi o ethernet opzionali si può usare dislocata da un altra parte rispetto a dove hai Homeassistant:https://github.com/Gio-dot/Z-Bee-Duo

Scusa, ma per quanto ad altri sembra essere chiaro, io non ho capito nemmeno dopo aver consultato la pagina al link che hai inserito.
Per me il coordinator che non ha bisogno di RPI che cerco è un coordinator che posso istruire per associare la pressione di un tasto di un telecomando all'accensione di una lampadina e questa associazione rimane memorizzata dentro al coordinator anche quando lo stacco dal suo RPI (senza zigbee2mqtt, deconz, ecc.), solo attaccando il coordinatore all'alimentazione, ad esempio ad un alimentatore USB.
Questo oggetto esiste?

**Reply by snakuzzo** [reply]
Non è scontato quello che hai detto...ovvero...

_"Per me il coordinator che non ha bisogno di RPI che cerco è un coordinator che posso istruire per associare la pressione di un tasto di un telecomando all'accensione di una lampadina e questa associazione rimane memorizzata dentro al coordinator"_

Dipende dal software che usi per gestire il coordinator HW...e quella funzionalità di cui parli si chiama "Binding"

**Reply by segaura** [reply]
Molto interessante, grazie, mi mancava questa parola chiave.
Ho letto un po' ora e vedo che non è una funzionalità del coordinator ma di due singoli oggetti che possono essere abbinati direttamente fra loro.  Nel mio caso ho telecomandi e faretti TRADFRI e dalla documentazione zigbee2mqtt è possibile fare il binding, ci proverò.

Sarebbe bello poter avere alcuni abbinamenti minimali per garantire un minimo di funzionalità in caso di problemi ai vari RPI o equivalenti, credo si salverebbero dei matrimoni.

Se invece qualcuno inventerà un coordinator nel quale possono essere memorizzate tutte le logiche di zigbee2mqtt per essere usate scollegandolo, penso sarà comunque una cosa molto interessante, è quello che succede con tanti tipi di domotica, per esempio KNX: l'installatore usa un software che ha solo lui e fa gli abbinamenti, poi si porta via il software e l'utente usa l'impianto senza cambiare più gli abbinamenti.

Per non esagerare con l'OT, ho ordinato anche io un Sonoff Zigbee Dongle Plus per sostituire il mio Conbee II e riporterò i miei feedback appena arriva.

**Reply by snakuzzo** [reply]
una soluzione del genere "io" non la adotterei mai nella vita...e comunque vorrebbe dire avere solo tecnologia zigbee e dovresti gestirla comunque tramite un hub. E se l'hub va giù? Il risultato sarebbe il medesimo.
Secondo me farsi troppe "pippe mentali" su questa cosa non ha senso. Il bello di avere un hub simil-home assistant è proprio quello di avere un punto unico con il quale gestire la propria domotica fatta di dispositivi di qualsiasi tecnologia.

**Reply by segaura** [reply]
> luzione del genere "io" non la adotterei mai nella vita...e comunque vorrebbe dire avere solo tecnologia zigbee e dovresti gestirla comunque tramite un hub. E se l'hub va giù? Il risultato sarebbe il medesimo.

Mi hai insegnato tu cosa sia il _binding_, ma ho la sensazione che non hai capito esattamente di cosa si tratti:

`It will work even when home automation software, Zigbee2MQTT or the coordinator is down.`

- due oggetti si parlano direttamente, un telecomando TRADFRI con una lampadina TRADFRI
- questo caso d'uso elementare, che magari ti occupa un solo pulsante di quel telecomando è garantito funzioni quando tutto il resto è giù, è quello che salva il matrimonio
- gli altri n-1 pulsanti del telecomando li gestisce home assistant e possono accendere una lampadina wifi, chiudere una tapparella z-wave, chiamare un webservice http/rest, ...

Nessuno obbliga nessuno a seguire nessuna strada, ma mi sembra che questo binding sia una opzione interessante.

**Reply by snakuzzo** [reply]
Guarda che io so perfettamente di cosa si tratta :)
Forse sei tu a non aver capito cosa intendevo.
Ti ho semplicemente esposto un mio pensiero che non si riferiva esclusivamente al binding (che è una cosa che può risultare utile), ma all'utilizzare tutto zigbee con un coordinator slegato da tutto il resto.

**Reply by segaura** [reply]
Sì, ha ragione, non avevo capito che avevi smesso di parlare di _binding_, ok.

**Comment by ster1um** [comment]
Tornando in tema sonoff plus va abbastanza bene.
Ho flashato l'ultima release sulla repo di zigbee2mqtt però quanto pare non è attiva la parte dell'amplificatore.

Domanda avere un firmware con flow control cosa comporta?
